receiver/awscontainerinsightreceiver/factory.go

```diff
@@ -54,6 +54,9 @@ const (
 
 	// Default locking resource name during EKS leader election
 	defaultLeaderLockName = "otel-container-insight-clusterleader"
+
+	// Don't enable EKS control plane metrics by default
+	defaultEnableControlPlaneMetrics = false
```

Comment:
You know what I just realized? We don't actually need to do this because the default value in the struct should just be false. 🤷 oh well. It's like this for the other ones